includes/Admin/Admin_Page.php

```diff
@@ -154,4 +157,14 @@ public function filter_plugin_action_links( $actions, $plugin_file ) {
 
 		return $actions;
 	}
+
+	/**
+	 * Render the results table templates in the footer.
+	 *
+	 * @since n.e.x.t
+	 */
+	public function admin_footer() {
+		require_once WP_PLUGIN_CHECK_PLUGIN_DIR_PATH . '/templates/results-table.php';
+		require_once WP_PLUGIN_CHECK_PLUGIN_DIR_PATH . '/templates/results-row.php';
```

Comment:
A few things here:
* These files should preferably only include the templates themselves, not the surrounding script tag. For that we should use the `wp_print_inline_script_tag()` function. To pass the output of these templates into the function, we can then use an output buffer.
* No need for `require_once`, this file doesn't contain code and is okay to be included multiple times.
```suggestion
		ob_start();
		require WP_PLUGIN_CHECK_PLUGIN_DIR_PATH . '/templates/results-table.php';
		$results_table_template = ob_get_clean();
		wp_print_inline_script_tag(
			$results_table_template,
			array(
				'id'   => 'tmpl-plugin-check-results-table',
				'type' => 'text/template',
			)
		);

		ob_start();
		require WP_PLUGIN_CHECK_PLUGIN_DIR_PATH . '/templates/results-row.php';
		$results_row_template = ob_get_clean();
		wp_print_inline_script_tag(
			$results_row_template,
			array(
				'id'   => 'tmpl-plugin-check-results-row',
				'type' => 'text/template',
			)
		);
```